Formula/lablgtk.rb

```diff
@@ -22,6 +17,9 @@ class Lablgtk < Formula
     sha256               x86_64_linux:   "30f9191323a86ef69a1794027fdf8c6ef6b49b97df043732038bb0c9423c1b2e"
   end
 
+  # GTK 2 is EOL: https://blog.gtk.org/2020/12/16/gtk-4-0/
+  deprecate! date: "2023-01-18", because: :unmaintained
```

Comment:
Was thinking that v3 should be its own formula (like other `*gtk*3` formulae) as it is listed on OPAM as [`lablgtk3`](https://opam.ocaml.org/packages/lablgtk3/). Alternatively, could just use `lablgtk` formula.

However, having trouble building as needs other OCaml libraries (e.g. `ocaml-cairo`, some other `dune` libraries not included in the formula, etc.)